src/components/Notifications.js

```diff
@@ -20,35 +102,85 @@ const Notifications = () => {
   };
 
   useEffect(() => {
-    messages.forEach(({ id, text, lifetime, type, isDismissed }) => {
-      if (isDismissed) {
-        closeSnackbar(id);
-        return;
+    function chooseIcon(icon, type) {
+      if (icon) {
+        return iconSelector(icon);
+      } else {
+        switch (type) {
+          case NotificationsTypes.SUCCESS:
```

Comment:
I think it makes sense to explicity write here the (alert) icon for success instead of relying on the default in case it changes, it changes here